src/Hosting/TestHost/src/TestWebSocket.cs

```diff
@@ -337,7 +332,7 @@ private void ThrowNoReceive()
         {
             if (_receiverClosed)
```

Comment:
The if/else is no longer needed.
```suggestion
ObjectDisposedException.ThrowIf(_receiverClosed, typeof(TestWebSocket));
// else _senderClosed
throw new IOException("The remote end closed the connection.", new ObjectDisposedException(typeof(TestWebSocket).FullName));
```

Reply:
Not sure I understand this change! :(

Reply:
I changed it